script/Arcueid/Day 2/02_00_ARC02_2_1.txt

```diff
@@ -31,7 +31,7 @@ The new teacher, who came to take Toyama-sensei's place, places her index finger
 [sha:90cc00f00f1b32e001cf7bf84ef29a02d38870b5]{
 -- Page 169, Offset 3041.
 -- 公園の外れで、体の半分が焼け落ちていたんだって」
-They found half of her body burnt to a crisp on the outskirts of the park."
+They found half of her body burnt to a crisp on the outskirts of a park."
```

Comment:
I think they're talking about a specific park here the one where Shiki and Arc meet.

Reply:
"the" park makes it sound like theres only one park in souya (which there might actually be considering we literally only see one) so i'll leave this up for now

Reply:
![image](https://user-images.githubusercontent.com/42716753/174256330-e8b3a037-e91c-48b7-ae7c-275b66daaa89.png)
There are at least two parks